src/modules/launcher/PowerLauncher/ViewModel/ResultsViewModel.cs

```diff
@@ -174,7 +174,7 @@ public void SelectPrevTabItem()
             {
                 // Tabbing backwards should highlight the last item of the previous row
                 SelectPrevResult();
-                SelectedItem.SelectLastContextButton();
+                SelectedItem?.SelectLastContextButton();
```

Comment:
Ahh, I hope to find another bug. Maybe I can try fixing it next time. That would be exiting. Thanks again!

Reply:
If you want to jump in, tons of things to help out on!